tket/src/Circuit/setters_and_getters.cpp

```diff
@@ -425,6 +425,15 @@ std::vector<std::optional<Edge>> Circuit::get_linear_out_edges(
   return outedges;
 }
 
+Edge Circuit::get_linear_edge(const Edge &e) const {
```

Comment:
Please can you add a basic test for this, including the Boolean case

Reply:
Sure thing!